### PR TITLE
feat(cli): Support react being defined in a higher package.json

### PR DESCRIPTION
**Pull request by tchock**

### CODE REVIEW

File: packages/cli/src/utils/getDependencyVersion.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import * as path from 'path';
+import * as fs from 'fs';
+
+export function getDependencyVersion(folderPath: string, dependencyName: string, fileSystem = fs) {
```
Comment (JohannesMerz) [inline packages/cli/src/utils/getDependencyVersion.ts:4]: you could have made use of nodes require mechanism

```
return require('react/package.json')?.version
```

Comment (JohannesMerz) [reply packages/cli/src/utils/getDependencyVersion.ts:4]: this should always find the actual used react package

**Review by JohannesMerz — CHANGES_REQUESTED** [review]
sth to discuss/consider

File: packages/cli/src/utils/getDependencyVersion.ts (hunk 1 above)
Comment (tchock) [reply packages/cli/src/utils/getDependencyVersion.ts:4]: No, this is about the react version

Comment (tchock) [reply packages/cli/src/utils/getDependencyVersion.ts:4]: And it will most probably find the react instance of the cli

Comment (JohannesMerz) [reply packages/cli/src/utils/getDependencyVersion.ts:4]: right, we also have execution path in our config

**Review by JohannesMerz — APPROVED** [review]

**Pull request merged**